### ISSUE TITLE
Clearml agent packages not properly installed/used?

### ISSUE THREAD

**Issue by ColdTeapot273K** [allegroai/clearml#545]
`clearml`==1.1.5
`clearml-agent`==1.1.1

Clearml agent executing task from a queue, package manager - `poetry`

```bash
...
Summary - installed python packages:
pip:
...
- 'clearml==1.1.6 # ClearML - Auto-Magical Experiment Manager, Version Control, and
    MLOps for AI'
...
Environment setup completed successfully

Starting Task Execution:

Traceback (most recent call last):
  File "path/to/my/script", line 3, in <module>
    from clearml import Task, Logger
ModuleNotFoundError: No module named 'clearml'
```

**Comment by jkhenning** [comment]
Hi @ColdTeapot273K ,

Can you share some more context from the agent's output?

**Comment by ColdTeapot273K** [comment]
@jkhenning 

Yep, the [full log](https://filen.io/d/2c94aad9-a6e4-4f6f-b9e6-b3dfdcf81eec#!rUa5h1FkzDnjWKJPBzzmQ7oQaJygo2iz)

**Comment by jkhenning** [comment]
> agent.python_binary = /home/acalabourdin/.pyenv/shims/python
> ...
> Creating virtualenv ds-614 in /home/acalabourdin/.clearml/venvs-builds/3.9/task_repository/ml_automation.git/.venv

It seems like you're running the agent from a virtualenv and not using the `--docker` option, which is not supported (this basically causes nested virtualenvs).

Can you try installing the agent in the system python and see if it works?
